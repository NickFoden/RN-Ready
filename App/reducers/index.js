import { combineReducers } from "redux";

import currencies from "./currencies";
import network from "./network";
import theme from "./theme";

export default combineReducers({
  currencies,
  network,
  theme
});
