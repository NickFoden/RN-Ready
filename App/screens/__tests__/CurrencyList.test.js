import React from "react";
import { shallow } from "enzyme";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import CurrencyList from "../CurrencyList";
import { initialState as currencyInitialState } from "../../reducers/currencies";
import { initialState as themeInitialState } from "../../reducers/theme";

const mockStore = configureStore([]);
it("succesfuly render currency list", () => {
  const navigation = { state: { params: { type: "quote" } } };
  const initialState = {
    currencies: currencyInitialState,
    theme: themeInitialState
  };
  const rendered = shallow(
    <Provider store={mockStore(initialState)}>
      <CurrencyList navigation={navigation} />
    </Provider>
  );
  expect(rendered.dive()).toMatchSnapshot();
});
