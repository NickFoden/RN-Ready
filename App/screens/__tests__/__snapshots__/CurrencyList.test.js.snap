// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`succesfuly render currency list 1`] = `ShallowWrapper {}`;
